### DISCUSSION TITLE
Is the pnpm-lock.yaml schema documented somewhere?

### DISCUSSION THREAD

**Discussion by n-insaidoo** [pnpm/pnpm#4952] [Q&A] [answered]
Hi, I'd like to be pointed to any online resource that documents the YAML schema of the pnpm-lock.yaml file. 
I'm looking for this resource to better understand the syntax below:

`18.0.0_react@18.2.0`

Appearing in a lockfile I previously generated in the following point 
```yaml
...
dependencies:
  axios: 0.27.2
  react: 18.2.0
  react-dom: 18.0.0_react@18.2.0
  uuid: 8.3.2
...
  /react-dom/18.0.0_react@18.2.0:
    resolution: {integrity: sha512-XqX7uzmFo0pUceWFCt7Gff6IyIMzFUn7QMZrbrQfGxtaxXZIcGQzoNpRLE3fQLnS4XzLLPMZX2T9TRcSrasicw==}
    peerDependencies:
      react: ^18.0.0
    dependencies:
      loose-envify: 1.4.0
      react: 18.2.0
      scheduler: 0.21.0
    dev: false
...
```

**Comment by zkochan** [comment, marked as answer]
Read this doc to understand why that suffix is added: https://pnpm.io/how-peers-are-resolved

It happens with dependencies that have peer dependnecies.
